.github/workflows/build.yml

```diff
@@ -3,23 +3,23 @@ name: Build
 on:
```

Comment:
commit ba83bac35a2733441be5e894ca84b3278c9b4244: how were you able to merge the PR when the CI fails?